Dockerfile

```diff
@@ -38,3 +38,8 @@ RUN apt-get update \
 COPY --from=composer /usr/bin/composer /usr/bin/composer
 
 FROM phpdoc_base as prod
+
+FROM prod as dev-pcov
```

Comment:
Did you see we have a dev target? 
https://github.com/phpDocumentor/phpDocumentor/blob/b48c54995079a58888d9150ca534992582ca6710/Dockerfile#L32

Reply:
Yes, but that is included in prod already (build stages are built from top to bottom always) I thought: let's take it from prod. But I can also extend it from dev if you'd like.

It can't be included in the dev stage as it would slow down the build

Reply:
Maybe we should turn them around?  So dev is latest? Reason for me the set prod as latest and accept the delay during the build was to mitigate the possibility for a dev build in production by accident. Not sure that's a valid reason

Reply:
Switching them around makes sense; I'll do that